src/main/java/com/revature/chronicle/controller/NoteController.java

```diff
@@ -112,4 +122,29 @@ public NoteController (NoteService ns, NoteRepo nr, TagRepo tr) {
         Optional<Note> targetNote = noteService.findById(id);
         return targetNote.map(note -> new ResponseEntity<>(note, HttpStatus.OK)).orElseGet(() -> new ResponseEntity<>(HttpStatus.BAD_REQUEST));
     }
+    
+    //Verify User Authorization
+    @PostMapping(path = "/whitelist/add/{noteId}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> addToWhitelist(@PathVariable(name="noteId") int id, @RequestParam("json") String json) throws JsonMappingException, JsonProcessingException {
```

Comment:
Instead of RequestParam, shouldn't it be RequestBody for the json param?  We aren't sending a list of users via the url.